sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/context/AzureContextUtils.java

```diff
@@ -12,34 +12,35 @@ private AzureContextUtils() {
 
     }
 
-    public static final String DEFAULT_TOKEN_CREDENTIAL_BEAN_NAME = "spring_cloud_azure_default_credential";
+    public static final String DEFAULT_TOKEN_CREDENTIAL_BEAN_NAME = "springCloudAzureDefaultCredential";
 
     public static final String EVENT_HUB_CLIENT_BUILDER_FACTORY_BEAN_NAME =
-        "EVENT_HUB_CLIENT_BUILDER_FACTORY_BEAN_NAME";
+        "springCloudAzureEventHubsClientBuilderFactory";
```

Comment:
Should we remove "s" and call it `springCloudAzureEventHubClientBuilderFactory` to stay consistent with the SDK class? same for the below ones.